relays/ethereum/src/ethereum_deploy_contract.rs

```diff
@@ -0,0 +1,136 @@
018080602001828103825260318152602001806108d66031913960400191505060405180910390fd5b6006810154608083015151156103fe57826040015181106103f1576040805162461bcd60e51b815260206004820152601960248201527f4f7665726c617070696e67207369676e616c7320666f756e6400000000000000604482015290519081900360640190fd5b5060608201516040830151015b60048201546005830154600384015467ffffffffffffffff90921691600260001960018316156101000201909116041561043f575060208401516001909101905b6040805160e0810182526001808252602088810151818401908152898501518486019081526080808c01516060870190815267ffffffffffffffff8a169187019190915260a0860188905260c086018a90528b51600090815260058552969096208551815460ff19169015151781559151938201939093559151600283015592518051929391926104d69260038501920190610770565b50608082015160048201805467ffffffffffffffff191667ffffffffffffffff90921691909117905560a0820151600582015560c09091015160069091015550509151600055505050565b60008281526005602052604090206002015483146105705760405162461bcd60e51b815260040180806020018281038252602f8152602001806108a7602f913960400191505060405180910390fd5b6002546000610580858585610731565b600081815260056020526040902060028281558101546001559091505b828214610637575060009081526005602052604090206001810154600682015460028301549192911415610632576005818101546000908152602091909152604090206003805467ffffffffffffffff198116600167ffffffffffffffff9283168101909216178255908201805461062b9260049291600261010092821615929092026000190116046107ee565b5050610637565b61059d565b505050505050565b61064761073b565b600080600080600060608751602089016040516020810160208101602081016020810160a08588886010600019fa61067e57600080fd5b93519251915190519351929b50909950975090955093505082159050610704578167ffffffffffffffff811180156106b557600080fd5b506040519080825280601f01601f1916602001820160405280156106e0576020820181803683370190505b50905087516020890160208301848184846011600019fa61070057600080fd5b5050505b6040805160a081018252968752602087019590955293850192909252606084015250608082015292915050565b6002549392505050565b6040518060a0016040528060008019168152602001600080191681526020016000815260200160008152602001606081525090565b828054600181600116156101000203166002900490600052602060002090601f016020900481019282601f106107b157805160ff19168380011785556107de565b828001600101855582156107de579182015b828111156107de5782518255916020019190600101906107c3565b506107ea929150610863565b5090565b828054600181600116156101000203166002900490600052602060002090601f016020900481019282601f1061082757805485556107de565b828001600101855582156107de57600052602060002091601f016020900482015b828111156107de578254825591600101919060010190610848565b61087d91905b808211156107ea5760008155600101610869565b9056fe4d697373696e6720706172656e74206865616465722066726f6d207468652073746f726167654d697373696e672066696e616c69747920746172676574206865616465722066726f6d207468652073746f726167654d697373696e672072657175697265642066696e616c6974792070726f6f6620666f7220706172656e7420686561646572a2646970667358221220920ca01b59940884722a766076b4f1d1d825eba446e530980bd310f776703dad64736f6c63430006060033")
```

Comment:
Given that this is going to change every time we update the contract we should probably read it from a file, no?